met-web/src/components/survey/listing/index.tsx

```diff
@@ -155,22 +165,112 @@ const SurveyListing = () => {
             key: 'comments_meta_data',
             numeric: true,
             disablePadding: false,
-            label: 'Responses',
+            label: 'Comments',
+            customStyle: { padding: 2 },
+            align: 'left',
+            hideSorticon: true,
             allowSort: false,
             getValue: (row: Survey) => {
-                if (!row.comments_meta_data.total) {
-                    return 0;
-                }
+                const isAuthorized = canViewPrivateEngagements || assignedEngagements.includes(Number(row.id));
 
-                const { total, pending } = row.comments_meta_data;
+                if (!isAuthorized) {
+                    return '';
+                }
                 return (
                     <MuiLink component={Link} to={`/surveys/${row.id}/comments`}>
-                        {`${total}`}
-                        {pending ? ` (${pending} New)` : ''}
+                        View All
                     </MuiLink>
                 );
             },
         },
+        {
+            key: 'comments_meta_data',
+            numeric: true,
+            disablePadding: false,
+            label: '',
```

Comment:
could this be duplicate code of engagement page?

Reply:
It's the same columns in the UI, with few differences in the key and the object they index, here it's row.comments_meta_data, engagement page is row.submissions_meta_data. Here the link is row.id, there it's row.surveys[0].id. So minor changes that might make it awkward to extract them into a common place.